pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -426,6 +426,11 @@ def resolve_training_type_plugin(self, training_type: TrainingTypePlugin) -> Tra
         if hasattr(training_type, 'num_nodes') and getattr(training_type, 'num_nodes') is None:
             training_type.num_nodes = self.num_nodes
 
+        # Automatically set sync_batchnorm if not None.
```

Comment:
```suggestion
        # Automatically set sync_batchnorm if None.
```

Perhaps I'm misunderstanding, but you're checking for the existence of this attr and if it is set to None as well?

Reply:
Ah yes, my mistake.